packages/runner-ct/tsconfig.json

```diff
@@ -55,10 +55,9 @@
     // "noResolve": true
   },
   "include": [
-    "./lib/*.ts"
-  ],
-  "files": [
-    "index.ts",
+    "./lib/*.ts",
```

Comment:
this `tsconfig` doesn't work for some reason. Need to figure this out

Reply:
I noticed this as well. I was going to suggest a different glob like `./lib/**/*.tsx?` but I am not sure vscode is even respecting this tsconfig at all - complaining about decorators.

Reply:
Not really it actually doesn't with with the `.ts` files also. I think I`ll try to fix this in separate PR